src/pages/index.js

```diff
@@ -0,0 +1,285 @@
+  const cardElement = cardTemplate.content
+    .querySelector(".card")
+    .cloneNode(true);
+  const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
+  const cardNameEl = cardElement.querySelector(".card__title");
+  const cardImageEl = cardElement.querySelector(".card__image");
+  const cardLikeBtn = cardElement.querySelector(".card__like-btn");
+  cardNameEl.textContent = data.name;
+  cardImageEl.src = data.link;
+  cardImageEl.alt = data.name;
+
+  cardDeleteBtn.addEventListener("click", (evt) =>
+    handleCardDelete(cardElement, data._id)
+  );
+
+  cardLikeBtn.addEventListener("click", (evt) => handleLike(evt, data._id));
+
+  cardImageEl.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalImageEl.src = data.link;
+    previewModalImageEl.alt = data.name;
+    previewModalCaptionEl.textContent = data.name;
+  });
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-btn_liked");
+  }
+  return cardElement;
+}
+
+function handleEscapeKey(evt) {
+  if (evt.key === "Escape") {
+    const openedModal = document.querySelector(".modal_opened");
+    if (openedModal) {
+      closeModal(openedModal);
+    }
+  }
+}
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keydown", handleEscapeKey);
+}
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keydown", handleEscapeKey);
+}
+function handleEditProfileSubmit(evt) {
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+  evt.preventDefault();
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      disableButton(
+        editModal.querySelector(settings.submitButtonSelector),
+        settings
+      );
+      setButtonText(submitBtn, false);
+    });
+}
+
+// Add card API
+function handleAddCardSubmit(evt) {
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+  evt.preventDefault();
+  const name = cardNameInput.value;
+  const link = cardLinkInput.value;
+  disableButton(cardModalSaveBtn, settings);
+  api
+    .addCard({ name, link })
+    .then((data) => {
+      const cardElement = getCardElement(data);
+      cardsList.prepend(cardElement);
+      closeModal(cardModal);
+      evt.target.reset();
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtn, false);
+    });
+}
+//avatar submit handler function
+function handleAvatarSubmit(evt) {
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+  evt.preventDefault();
+  api
+    .editAvatarInfo(avatarInput.value)
+    .then((data) => {
+      profileAvatar.src = data.avatar;
+      avatarInput.value = data.avatar;
```

Comment:
` avatarInput.value = data.avatar;`

This is not needed here because you reset the form anyway